nix/gnutls.nix

```diff
@@ -0,0 +1,40 @@
+{ pkgs }:
+pkgs.stdenv.mkDerivation rec {
+  pname = "gnutls";
+  version = "3.7.3";
+
+  src = fetchTarball {
+    url =
+      "https://s3-us-west-2.amazonaws.com/s2n-public-test-dependencies/2022-01-18_gnutls-3.7.3.tar.xz";
+    sha256 = "sha256:07rk09hz138m0l5vrvymyj2z2is92mwykqzzf81d8xgbpn2dyapc";
+  };
+
+  buildInputs = [
+    (import ./nettle.nix { pkgs = pkgs; })
```

Comment:
Might be better to move this to a variable at the top since you're referring to it multiple times.